acme_tiny.py

```diff
@@ -5,8 +5,13 @@
 except ImportError:
     from urllib2 import urlopen # Python 2
 
-#DEFAULT_CA = "https://acme-staging.api.letsencrypt.org"
-DEFAULT_CA = "https://acme-v01.api.letsencrypt.org"
+STAGING_FILE = '/tmp/acme_tiny_staging'
+
+if 'ACME_TINY_STAGING' in os.environ or os.path.isfile(STAGING_FILE):
+    DEFAULT_CA = "https://acme-staging.api.letsencrypt.org"
+    print 'Work with staging server'
```

Comment:
I "print" to not move block after logger definition.
I can remove the print, or move it after logger block and use it.